src/Storage/Storage/Storage.csproj

```diff
@@ -13,15 +13,22 @@
     <RootNamespace>Microsoft.WindowsAzure.Commands.$(PsModuleName)</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Cosmos.Table" Version="0.10.1-preview" />
     <PackageReference Include="System.Security.Permissions" Version="4.5.0" />
-    <PackageReference Include="WindowsAzure.Storage" Version="9.3.0" />
-    <PackageReference Include="Microsoft.Azure.Storage.DataMovement" Version="0.8.1" />
+    <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="9.4.2" />
+    <PackageReference Include="Microsoft.Azure.Storage.File" Version="9.4.2" />
+    <PackageReference Include="Microsoft.Azure.Storage.Queue" Version="9.4.2" />
+    <PackageReference Include="Microsoft.Azure.KeyVault.Core" Version="2.0.4" />
```

Comment:
These Microsoft.Azure.Storage.* assemblies look like they won't be any problem

Reply:
Thanks for the review!